src/SFA.DAS.Admin.Roatp.Web/Controllers/AddProvider/ProviderDetailsController.cs

```diff
@@ -25,10 +25,23 @@ public IActionResult Index()
             TradingName = sessionModel.TradingName ?? NotApplicableValue,
             CompanyNumber = sessionModel.CompanyNumber ?? NotApplicableValue,
             CharityNumber = sessionModel.CharityNumber ?? NotApplicableValue,
-            AddProviderRouteUrl = Url.RouteUrl(RouteNames.ProviderDetails)!,
             SelectProviderUrl = Url.RouteUrl(RouteNames.AddProvider)!
         };
 
         return View(model);
     }
+
+    [HttpPost]
+    public IActionResult Index(ProviderDetailsViewModel model)
```

Comment:
Not sure why you decided to do this? We can keep the selection in the session, there was no need to clear if off.